lib/decrypter.rb

```diff
@@ -0,0 +1,43 @@
+class Decrypter
+  ALPHA = ("a".."z").to_a << " "
+  attr_reader :ciphertext, :key, :date
+  def initialize(ciphertext, key, date)
+    @ciphertext = ciphertext
+    @key = key
+    @date = date
+  end
+
+  def key_maker
+    split_key = [
+      key[0, 2].to_i,
+      key[1, 2].to_i,
+      key[2, 2].to_i,
+      key[3, 2].to_i,
+    ]
+
+    modifier = (date.to_i ** 2).to_s[-4, 4]
+
+    4.times { |time| split_key[time] += modifier[time].to_i}
+
+    split_key
+  end
```

Comment:
`Decrypter#key_maker` and `Encrypter#key_maker` look similar. How would we refactor this so that we're not repeating ourselves?